src/App.tsx

```diff
@@ -68,14 +67,11 @@ export default function App({ history }: AppProps) {
                     <SentryRoute exact path="/register">
                       <RegisterPage />
                     </SentryRoute>
-                    <SentryRoute path="/room/:roomId?">
```

Comment:
(and also still keep a `/room/:roomId` route with a `Redirect` to the simplified URL, for backwards compatibility)

Reply:
I don't think that is necessary? `getUrlParams()` should already handle `/room/...` URLs

Reply:
Oh, I see, the `*` route would cause `/room` to be routed to the room page, even though it looks like it needs to be a special case. I didn't catch that at first.

Reply:
No worries :)